Update supported databases in v19

GuptaManan100 opened the pull request:
### Description

This PR updates the supported databases in v19. We also now list the databases that we support for importing.

GuptaManan100 committed:
docs: updated supported databases in v19

Signed-off-by: Manan Gupta <manan@planetscale.com>

GuptaManan100 commented on content/en/docs/19.0/overview/supported-databases.md:
```diff
@@ -4,15 +4,23 @@ weight: 2
-{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended.{{< /info >}}
+- [MySQL](https://www.mysql.com/) version 5.7 to 8.0
+- [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) version 5.7 to 8.0
+- [MariaDB](https://mariadb.com)
```
I am unsure of the versions of MariaDB that we allow support from

deepthi replied:
Good question. @mattlord what should we say here?

mattlord replied:
I'd say MariaDB 10.10+. If someone reports a failure with a later version then I'd consider it a bug.

GuptaManan100 replied:
Awesome possum. I'll make the change!

GuptaManan100 commented on content/en/docs/19.0/overview/supported-databases.md:
```diff
@@ -4,15 +4,23 @@ weight: 2
-{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended.{{< /info >}}
+- [MySQL](https://www.mysql.com/) version 5.7 to 8.0
+- [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) version 5.7 to 8.0
+- [MariaDB](https://mariadb.com)
```
Do we want to list anything else like RDS, etc?

mattlord replied:
I don't think so. RDS has MySQL 8.0 e.g. so there's some implicit support there.

mattlord commented on content/en/docs/19.0/overview/supported-databases.md:
```diff
@@ -4,15 +4,23 @@ weight: 2
 The [VTGate](../../concepts/vtgate/) proxy server advertises its version as MySQL 8.0.
 
-## MySQL versions 5.7 to 8.0
+### MySQL versions 8.0
```
We can make versions singular.

mattlord reviewed (commented):
IMO we should also add warnings to the v17 and v18 docs that MySQL 5.7 support is EOL in v19.

deepthi commented on content/en/docs/19.0/overview/supported-databases.md:
```diff
@@ -4,15 +4,23 @@ weight: 2
-## MySQL versions 5.7 to 8.0
+### MySQL versions 8.0
+
+Vitess supports the core features of MySQL versions 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 8.0.
```
We can probably drop "versions" altogether, both in the heading and in the description.

GuptaManan100 committed:
feat: address review comments

Signed-off-by: Manan Gupta <manan@planetscale.com>

GuptaManan100 commented:
I removed the `Versions` as you two pointed out. I have also added a line to both v18.0 and v17.0 docs to say that MySQL 5.7 is EOL and users should upgrade to 8.0 before upgrading to v19.

GuptaManan100 committed:
feat: specify mariadb versions supported

Signed-off-by: Manan Gupta <manan@planetscale.com>

deepthi commented on content/en/docs/17.0/overview/supported-databases.md:
```diff
@@ -12,7 +12,7 @@ The [VTGate](../../concepts/vtgate/) proxy server advertises its version as MySQ
 Vitess supports the core features of MySQL versions 5.7 to 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 5.7 to 8.0.
 
-{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended.{{< /info >}}
+{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended. MySQL version 5.7 is EOL in v19 and users are requested to upgrade to 8.0 before upgrading to Vitess v19. {{< /info >}}
```
I would make this stronger - active voice vs passive voice.
"users should upgrade to MySQL 8 before upgrading to Vitess v19."

8 versus 8.0 for consistency with previous sentence.

deepthi commented on content/en/docs/18.0/overview/supported-databases.md:
```diff
@@ -12,7 +12,7 @@ The [VTGate](../../concepts/vtgate/) proxy server advertises its version as MySQ
 Vitess supports the core features of MySQL versions 5.7 to 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 5.7 to 8.0.
 
-{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended.{{< /info >}}
+{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended. MySQL version 5.7 is EOL in v19 and users are requested to upgrade to 8.0 before upgrading to Vitess v19. {{< /info >}}
```
I would make this stronger - active voice vs passive voice.
"users should upgrade to MySQL 8 before upgrading to Vitess v19."

8 versus 8.0 for consistency with previous sentence.

GuptaManan100 committed:
feat: address review comments

Signed-off-by: Manan Gupta <manan@planetscale.com>

mattlord commented on content/en/docs/17.0/overview/supported-databases.md:
```diff
@@ -12,7 +12,7 @@ The [VTGate](../../concepts/vtgate/) proxy server advertises its version as MySQ
 Vitess supports the core features of MySQL versions 5.7 to 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 5.7 to 8.0.
 
-{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended.{{< /info >}}
+{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended. MySQL version 5.7 is EOL in v19 and users should upgrade to MySQL 8 before upgrading to Vitess v19. {{< /info >}}
```
Might be worth a separate warning block, but up to you. This is fine too IMO.

GuptaManan100 replied:
Done

mattlord commented on content/en/docs/19.0/overview/supported-databases.md:
```diff
@@ -4,15 +4,23 @@ weight: 2
+## Supported Databases For Imports
 
-Vitess supports the core features of MySQL versions 5.7 to 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 5.7 to 8.0.
+Vitess supports importing from a wide range of databases that include - 
```
```suggestion
Vitess supports importing from a wide range of databases that include:
```

mattlord reviewed (approved):
LGTM! I only had minor notes that you can resolve as you feel is best. Thanks!

GuptaManan100 committed:
feat: make separate info blocks

Signed-off-by: Manan Gupta <manan@planetscale.com>

GuptaManan100 committed:
feat: use semi-colon instead

Signed-off-by: Manan Gupta <manan@planetscale.com>

GuptaManan100 committed:
feat: use warning instead of info

Signed-off-by: Manan Gupta <manan@planetscale.com>
